server.py

```diff
@@ -154,26 +149,21 @@ def render_document(self, t_file, context):
         Args:
             t_file: template file
             context: (dict) template variables
-
         Returns:
             (str) downloaded file
-
         """
-
+        odt_template = BytesIO(t_file)
         engine = Renderer()
-        rendered = engine.render(t_file, **context)
+        rendered = engine.render(odt_template, **context)
         return "%s%s" % (S3_PUBLIC_URL, self.save_document(rendered))
```

Comment:
revert back, t_file will be a BytesIO object after your modification of lines above.